includes/admin/tools/views/html-admin-page-data.php

```diff
@@ -54,9 +54,11 @@
 					<span class="tools-form-dropdown tools-form-dropdown-recount-form" style="display: none">
 						<?php
 						$args = array(
+							'class'   => 'tools-form-dropdown-recount-form-select',
 							'name'   => 'form_id',
 							'number' => - 1,
 							'chosen' => true,
+							'placeholder' => 'Select Form',
```

Comment:
@raftaar1191 Why we have text without  i18n fn?

Reply:
Fixing that